Felienne/hedy
# [BUG FIX] Country correctly removed from highscores when removed from profile

**Description**
In this PR we fix a bug where a country update on the user profile settings would only be updated on the highscores if it actually contained a country. When removing the country from the profile settings this didn't affect the highscore settings: which is of course wrong. We fix this by always updating the country settings on the highscores, independent of the profile update.

**Fixes**
This PR fixes #3000.

**How to test**
Make sure you have an account with a public profile and achievements (and thereby automatically a highscore setting). Verify that your profile country is shown on the highscores page. When removing this country from your profile settings this should also affect the highscore settings.
